graphql/queries/getMe.graphql

```diff
@@ -0,0 +1,21 @@
+query getMember {
```

Comment:
@csell5 I'd like to start to push these shared queries off so that each can be imported rather than duplicated, but it's giving me issues when I go that route, `ApolloError: Network error: Error writing result to store for query:` and whole app goes down the crapper. Be great to get your 👀 up in here to help me spot what I'm missing.